cmd/main.go

```diff
@@ -70,35 +78,36 @@ func main() { //nolint:funlen // main function needs to initialize many object
 		"ID for the controller leader election.")
 	flag.IntVar(&metricsPort, "metricsPort", defaultMetricsPort, "Port number for metrics endpoint.")
 
+	// setup k8s ctrl logger
+	logLevel, err := zapcore.ParseLevel(envConfigs.LogLevel)
+	if err != nil {
+		setupLog.Error(err, "unable to parse log level")
+		os.Exit(1)
+	}
 	opts := k8szap.Options{
-		Development: true,
+		Development: false,
```

Comment:
- What's the effect of setting this to true/false?
- Does it make sense to make this configurable too?